pkg/util/utils.go

```diff
@@ -76,3 +79,10 @@ func RetryAfter(attempts int, callback func() error, d time.Duration) (err error
 	}
 	return err
 }
+
+func GetLocalkubeDownloadURL(version string, filename string) string {
+	if strings.HasPrefix(version, "http://") {
```

Comment:
Use `url.Parse` to check it's a valid URL, which will include file URIs too which is nice.